server/utils/checkYourAnswersUtils.test.ts

```diff
@@ -182,6 +182,44 @@ describe('checkYourAnswersUtils', () => {
         task: 'confirm-eligibility',
         pageKey: 'page1',
         questions: mockedConfirmEligibilityQuestion,
+        checkYourAnswersPage: true,
+      })
+
+      expect(items).toEqual(expected)
+    })
+
+    it('adds each question and answer to the items array if checkYourAnswersPage flag is false', () => {
+      const mockedConfirmEligibilityQuestion = {
+        'confirm-eligibility': {
+          page1: {
+            question1: { question: 'A question', answers: { yes: 'Yes', no: 'No' } },
+            question2: { question: 'Another question' },
+          },
+        },
+      } as unknown as Questions
+
+      jest.spyOn(checkYourAnswersUtils, 'getPage').mockReturnValueOnce(jest.fn())
+
+      const items: Array<SummaryListItem> = []
+
+      const expected = [
+        {
+          question: 'A question',
+          answer: 'No',
+        },
+        {
+          question: 'Another question',
+          answer: 'an answer',
+        },
+      ]
+
+      addPageAnswersToItemsArray({
+        items,
+        application: mockApplication,
+        task: 'confirm-eligibility',
+        pageKey: 'page1',
+        questions: mockedConfirmEligibilityQuestion,
+        checkYourAnswersPage: false,
```

Comment:
do you think it might be clearer if we gave the "for the document" pathway equal status to the "for check your answers"?

i.e. rather than a boolean on the `checkYourAnswersPage` param, you do something like:

`outputFormat: 'checkYourAnswers' | 'document'`

Reply:
I like this idea!

Reply:
> i.e. rather than a boolean on the checkYourAnswersPage param, you do something like:

> outputFormat: 'checkYourAnswers' | 'document'

this sounds like a good idea, I've added a new commit introducing an `outputFormat`